docs/topics/intellij-idea-plugin-run-configuration.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-run-configuration_{context}"]
+= Configuring a run configuration
+
+You can configure multiple run configurations to run against each project you import to IntelliJ IDEA.
+
+.Procedure
+
+. In *Projects* view, click the project you want to analyze.
```

Comment:
1. "the" -- OK. I've one back and forth with using the definite article with "view" over the last decade, but I'll go with it.
2. The GUI object is named Products, so the use of bold is correct.  https://redhat-documentation.github.io/supplementary-style-guide/#user-interface-elements
3. "Activity" in "The Activity bar" is not bolded because I am using the generic term. The interface does not have labeled sections.

Reply:
RE 3, ping me on chat. I'm not sure what you mean by "generic term".